l10n/en/banners/fundraising.ftl

```diff
@@ -0,0 +1,16 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/accounts/
```

Comment:
Copy pasta on the URL?

```suggestion
### URL: https://www-dev.allizom.org/
```

Reply:
🤦 pasted from another file and didn't actually fix it. I'm an idiot.